en/docs/get-started/integration-quick-start-guide.md

```diff
@@ -28,8 +28,8 @@ Most healthcare centers have a system that is used to make doctor appointments.
 <a href="{{base_path}}/assets/img/integrate/quick-start-guide/mi-quick-start-guide.png"><img src="{{base_path}}/assets/img/integrate/quick-start-guide/mi-quick-start-guide.png" width="1600" height="600"></a>
 
 
-    !!! Tip
-        You may export` <mi-qsg-home>/HealthcareIntegrationProject` to Integration Studio to view the project structure.
+!!! Tip
+    You may export `<mi-qsg-home>/HealthcareIntegrationProject` to Integration Studio to view the project structure.
```

Comment:
`<mi-qsg-home>` should be `<MI_QSG_HOME>` in order to make sure that it follows the command standard in the documentation.

For example:

- `<HELM_HOME>` is used in this page https://apim.docs.wso2.com/en/latest/install-and-setup/install/deploying-api-manager-with-kubernetes-or-openshift-resources/ it is 
- `<IS_HOME>` is used in this page https://apim.docs.wso2.com/en/latest/design/api-security/oauth2/grant-types/jwt-grant/#step-2-configure-an-identity-provider-and-a-service-provider-in-wso2-api-m